src/pages/index.tsx

```diff
@@ -558,241 +334,296 @@ const HomePage = ({
+            <StyledTitleCardList
+              content={codeExamples}
+              clickHandler={toggleCodeExample}
+              headerKey="page-index-developers-code-examples"
+              icon="code"
+              isCode
+              border="1px"
+              borderColor="text"
+              boxShadow={cardBoxShadow}
+              maxWidth={{ lg: "624px" }}
+              ml={{ lg: 16 }}
+            />
+          </Box>
+          <FeatureContent>
+            <SectionHeading>
               <Translation id="page-index-developers" />
-            </StyledH2>
-            <Subtitle>
+            </SectionHeading>
+            <SectionDecription>
               <Translation id="page-index-developers-description" />
-            </Subtitle>
-            <ButtonRow>
-              <ButtonLink to="/developers/">
-                <Translation id="page-index-developers-button" />
-              </ButtonLink>
-            </ButtonRow>
-          </LeftColumnContent>
-        </FeatureContent>
-        <CodeboxModal
-          isOpen={isModalOpen}
-          setIsOpen={setModalOpen}
-          title={codeExamples[activeCode].title}
-        >
-          <Codeblock
-            codeLanguage={codeExamples[activeCode].codeLanguage}
-            allowCollapse={false}
-            fromHomepage
+            </SectionDecription>
+            <ButtonLinkRow
+              firstButton={{
+                to: "/dapps/?category=technology",
+                child: <Translation id="page-index-developers-button" />,
+              }}
+            />
+          </FeatureContent>
+          <StyledCodeModal
+            isOpen={isModalOpen}
+            setIsOpen={setModalOpen}
+            title={codeExamples[activeCode].title}
+            sx={{
+              ".modal-component-container": {
+                padding: 0,
+                left: 0,
+                right: 0,
+                bottom: 0,
+                top: "50%",
+              },
+              ".modal-component": {
+                maxWidth: "100%",
+                maxHeight: "50%",
+                padding: 0,
+              },
+              ".modal-component-content": {
+                marginTop: "3rem",
+                width: "100%",
+                overflow: "auto",
+              },
+            }}
           >
-            {codeExamples[activeCode].code}
-          </Codeblock>
-        </CodeboxModal>
-      </DeveloperContainer>
-      <StyledGrayContainer>
-        <StyledContent>
-          <h2>
+            <Codeblock
+              codeLanguage={codeExamples[activeCode].codeLanguage}
+              allowCollapse={false}
+              fromHomepage
+            >
+              {codeExamples[activeCode].code}
+            </Codeblock>
+          </StyledCodeModal>
+        </Row>
+      </MainSectionContainer>
+      {/* Eth Today Section */}
+      <GrayContainer>
+        <ContentBox>
+          <SectionHeading mt={undefined} mb={undefined} fontFamily={undefined}>
```

Comment:
Leveraging the `SectionHeading` component in replacing the `h2` tag that was here and in the next section.

Since `mt`, `mb`, and `fontFamily` are defined in the component and overriding other styles specific to the `h2` that are more preferable here, setting the values to `undefined` essentially cancels the overrides while still gaining the reuseability of the component.

Is this OK?

Reply:
Don't like it tbh 😄

Lets do this for now. `heading` is the old theme font family.
```suggestion
          <SectionHeading mt={12} mb={8} fontFamily="heading">
```